### PR TITLE
Xcode 9beta2 information

### PR DESCRIPTION
**Pull request by solarce**
Xcode 9 beta 2 information
```
$ xcodebuild -version

Xcode 9.0

Build version 9M137d
```

### CODE REVIEW

File: _data/xcodes.yml [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+      - iOS 8.4
+      - iOS 9.0
+      - iOS 9.1
+      - iOS 9.2
+      - iOS 9.3
+      - iOS 10.0
+      - iOS 10.1
+      - iOS 10.2
+      - iOS 10.3
+      - ios 10.11
```
Comment (emdantrim) [inline _data/xcodes.yml]: this one isn't capitalized `iOS` but the rest are. also, do the versions really go from 10.3 to 10.11?

File: _data/xcodes.yml [hunk 2]
```diff
@@ -1,4 +1,43 @@
 osx_images:
+  - image: xcode9
+    xcode: "9.0"
+    osx_version: "10.12"
+    xcode_full_version: "9.0"
```
Comment (emdantrim) [inline _data/xcodes.yml]: this should say "9.0 Beta 3"

File: _data/xcodes.yml [hunk 3]
```diff
@@ -1,4 +1,43 @@
 osx_images:
+  - image: xcode9
+    xcode: "9.0"
+    osx_version: "10.12"
+    xcode_full_version: "9 beta 2"
```
Comment (emdantrim) [inline _data/xcodes.yml]: I'd prefer a capital B just to be in line with how it looks for everything else, but it's not super important. :b:

Comment (solarce) [inline _data/xcodes.yml]: Since it's named **9 beta 2** on https://developer.apple.com/download/ I thought following that would make the most sense

**Pull request merged**